src/index.html

```diff
@@ -11,8 +11,61 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The href attribute is pointing to a SCSS file. Typically, you should link to a CSS file in HTML. Ensure that the SCSS is compiled to CSS and link to the resulting CSS file instead.